src/Npgsql/Npgsql.csproj

```diff
@@ -8,7 +7,7 @@
     <PackageTags>npgsql postgresql postgres ado ado.net database sql</PackageTags>
     <VersionPrefix>4.0.0</VersionPrefix>
     <LangVersion>latest</LangVersion>
-    <TargetFrameworks>net45;net451;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
I only now noticed that `Microsoft.Extensions.Primitives` doesn't support pre-netstandard2.0, which is another point against it in the context of Npgsql.

It's not that I care so much about supporting net45/net451 specifically, but continuing to support these platforms doesn't really cost us anything at the moment, so I wouldn't want to drop support for them just because of range string parsing...

Reply:
Same as above, meant to exclude that again.